.fleet/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "nodejs.editor.formatOnSave.prettier.mode": "Disabled"
+}
```

Comment:
I was debating if I should've added this in case others used fleet on the website as Fleet automatically formats entire documents on save. This would've affected every document since it seems like none of them have actually been formatted with prettier despite it being part of the project.

Reply:
That makes sense - unfortunately I think you're right, that none of us really used prettier. I suspect it was part of a gatsby starter that was used to bootstrap the project, then really not considered past that point. At some point we could probably afford to either remove the config, or bring it more in line with how we actually write code.